index.html

```diff
@@ -1,13 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mobile portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+    <header class="header">
+               <a href="#" class="logo">C.A.</a>
+               <img class="menuIcone" src="./Images/Union.png" alt="Union Image">
+               </a>
+                <!-- Hamburger icon -->
+                <input class="side-menu" type="checkbox" id="side-menu" />
+                <label class="hamb" for="side-menu"><span class="hamb-line"></span></label>
+               <!-- Menu -->
+               <nav class="nav">
+                <ul class="menu">
+                    <li><a href="#">Porfolio</a></li>
+                    <li><a href="#">About</a> </li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
```

Comment:
- [ ] kindly consider not displaying this as its making your design look messy and it not required for this milestone removing it will help you to implement well the required change
- hint : 
```
.menu { 
display: none
}
```